examples/py310/doc-examples-python-apis.py

```diff
@@ -166,6 +169,84 @@ async def example_API_TopicPublish(topic_client: TopicClientAsync):
 
 # end example
 
+async def example_API_InstantiateVectorClient():
+    PreviewVectorIndexClientAsync(
+        VectorIndexConfigurations.Default.latest(), CredentialProvider.from_environment_variable("MOMENTO_API_KEY")
+    )
+
+
+# end example
+
+async def example_API_CreateIndex(vector_client: PreviewVectorIndexClientAsync):
+    response = await vector_client.create_index("test-index", 2)
+    match response:
+        case CreateIndex.Success:
```

Comment:
Python's match statements are kind of odd. Each of the case statements needs to be an instance of the object:
```suggestion
        case CreateIndex.Success():
```

Reply:
Good catch.